tests/test_data/test_autoaug.py

```diff
@@ -0,0 +1,326 @@
+        _check_keys(results, results_translated)
+        # check image
+        _check_img_seg(results, results_translated,
+                       results.get('img_fields', ['img']), offset,
+                       img_fill_val, axis)
+        # check segmentation map
+        _check_img_seg(results, results_translated,
+                       results.get('seg_fields', []), offset, seg_ignore_label,
+                       axis)
+        # check masks and bboxes
+        _check_bbox_mask(results, results_translated, offset, axis, min_size)
+
+    # test case when level=0 (without translate aug)
+    img_fill_val = (104, 116, 124)
+    seg_ignore_label = 255
+    transform = dict(
+        type='Translate',
+        level=0,
+        prob=1.0,
+        fill_val=img_fill_val,
+        seg_ignore_label=seg_ignore_label)
+    translate_module = build_from_cfg(transform, PIPELINES)
+    results_wo_translate = translate_module(copy.deepcopy(results))
+    check_translate(
+        copy.deepcopy(results),
+        results_wo_translate,
+        0,
+        img_fill_val,
+        seg_ignore_label,
+        'x',
+    )
+
+    # test case when level>0 and translate along x-axis (left shift).
+    transform = dict(
+        type='Translate',
+        level=8,
+        prob=1.0,
+        fill_val=img_fill_val,
+        seg_ignore_label=seg_ignore_label,
+        axis='x')
+    translate_module = build_from_cfg(transform, PIPELINES)
+    offset = translate_module.offset
+    results_translated = translate_module(
+        copy.deepcopy(results), neg_offset_prob=1.0)
+    check_translate(
+        copy.deepcopy(results),
+        results_translated,
+        -offset,
+        img_fill_val,
+        seg_ignore_label,
+        'x',
+    )
+
+    # test case when level>0 and translate along x-axis (right shift).
+    results_translated = translate_module(
+        copy.deepcopy(results), neg_offset_prob=0.0)
+    check_translate(
+        copy.deepcopy(results),
+        results_translated,
+        offset,
+        img_fill_val,
+        seg_ignore_label,
+        'x',
+    )
+
+    # test case when level>0 and translate along y-axis (top shift).
+    transform = dict(
+        type='Translate',
+        level=10,
+        prob=1.0,
+        fill_val=img_fill_val,
+        seg_ignore_label=seg_ignore_label,
+        axis='y')
+    translate_module = build_from_cfg(transform, PIPELINES)
+    offset = translate_module.offset
+    results_translated = translate_module(
+        copy.deepcopy(results), neg_offset_prob=1.0)
+    check_translate(
+        copy.deepcopy(results), results_translated, -offset, img_fill_val,
+        seg_ignore_label, 'y')
+
+    # test case when level>0 and translate along y-axis (bottom shift).
+    results_translated = translate_module(
+        copy.deepcopy(results), neg_offset_prob=0.0)
+    check_translate(
+        copy.deepcopy(results), results_translated, offset, img_fill_val,
+        seg_ignore_label, 'y')
```

Comment:
The test only tests `BitmapMask` but miss `PolygonMask`.

Reply:
Hi, how can I convert a mask with type np.ndarray to 'polygons', which can be allowed to pass to 'PolygonMasks'

Reply:
Currently, we don't support convert BitMap to Polygons.
You can also add unit tests for Polygon as follows:
```
    with pytest.raises(NotImplementedError):
        trans(polygon_masks)
```
Try to increase the cover rate of auto_test.py, now is 80%.
You can get cover rate by the command, it also tells you which lines are not tested
```
pytest tests/test_data/test_autoaug.py --cov mmdet/datasets/pipelines/ --cov-report term-missing
```

Reply:
Hi, the cover rate of 'Translate' now is 88%, and the uncovered case includes two aspects: class 'AutoAugment' which will be tested in the future version, and 'PolygonMasks' (not supported currently) case which will be ignored by keyword 'pass'.